src/containers/tud/utils/index.js

```diff
@@ -518,6 +519,38 @@ function writeToCsvFile(
   FS.saveAs(blob, outputFileName);
 }
 
+function exportSummarizedDataToCsvFile(
```

Comment:
Why didn't you do this in the server?

Reply:
Not computation expensive, should be safe to do from browser